### PR TITLE
Add token component

### PR DESCRIPTION
**Pull request by pivilartisant**
Screenshot 2023-08-01 at 10 18 23

### CODE REVIEW

File: src/components/Token/Token.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      className={`w-full flex justify-between items-center text-neutral ${customClass}`}
+      {...rest}
+    >
+      <div className="flex w-fit gap-2 items-center">
+        {logo}
+        <div className="flex flex-col ">
+          <p className="mas-menu-active">{`${name} (${symbol})`} </p>
+          <span className="flex items-center gap-2">
+            <p className="mas-menu">{formattedBalance}</p>
+            {/* see wallet for tooltip size */}
```
Comment (mazmassa) [inline src/components/Token/Token.tsx:41]:
```suggestion
```

File: src/components/Token/Token.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+export interface ToggleProps extends ComponentPropsWithoutRef<'div'> {
+  token: ITokenData;
+  tooltipContent?: string;
+  customClass?: string;
+  onDelete?: () => void;
+}
+
+export function Token({ ...props }) {
+  const {
+    // token object that BE will send
```
Comment (mazmassa) [inline src/components/Token/Token.tsx:23]:
```suggestion
```

File: src/components/Token/Token.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      {...rest}
+    >
+      <div className="flex w-fit gap-2 items-center">
+        {logo}
+        <div className="flex flex-col ">
+          <p className="mas-menu-active">{`${name} (${symbol})`} </p>
+          <span className="flex items-center gap-2">
+            <p className="mas-menu">{formattedBalance}</p>
+            {/* see wallet for tooltip size */}
+            <Tooltip content={unformattedBalance} />
```
Comment (mazmassa) [inline src/components/Token/Token.tsx:42]:
```suggestion
            <Tooltip content={rawBalance} />
```

File: src/components/Token/Token.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  token: ITokenData;
+  tooltipContent?: string;
+  customClass?: string;
+  onDelete?: () => void;
+}
+
+export function Token({ ...props }) {
+  const {
+    // token object that BE will send
+    token: { logo, name, symbol, formattedBalance, unformattedBalance },
```
Comment (mazmassa) [inline src/components/Token/Token.tsx:24]:
```suggestion
    token: { logo, name, symbol, formattedBalance, rawBalance },
```

**Review by maxime-carabina — APPROVED** [review]
LGTM 🎉

**Review by mazmassa — APPROVED** [review]

**Pull request merged**